README.md

```diff
@@ -1,130 +1,89 @@
 
-#### Automated version with Grunt <https://github.com/gdg-x/zeppelin-grunt>
+For a detailed walkthrough, watch the project presentation from [GDG[x] Townhall meeting](http://www.youtube.com/watch?v=xYmhheoLjcI). Slides available [here](https://docs.google.com/presentation/d/19aM7yNl_orDaCNND5LpCY3fShb6PyMltnzYfKvV8R_8/edit?usp=sharing)
 
-### Features
-* Easy to setup
-* Simple and responsive design
-* Integrated speakers and sessions management
-* SVG icons
-* SEO friendly
 
+## Local development ##
 
-### Quick-start guide
-1. [Fork](https://github.com/gdg-x/zeppelin/fork) this repo
-2. Clone locally
-3. Update ```_config.yml```
-4. Select what content blocks do you need
-5. Push changes to ```gh-pages``` branch
-6. Enjoy your awesome DevFest site at ```http://[your github name].github.io/zeppelin/```
-
-Or watch project presentation from [GDG[x] Townhall meeting](http://www.youtube.com/watch?v=xYmhheoLjcI). Slides available [here](https://docs.google.com/presentation/d/19aM7yNl_orDaCNND5LpCY3fShb6PyMltnzYfKvV8R_8/edit?usp=sharing)
-
+Check if you have [all requirements for local environment](http://jekyllrb.com/docs/installation/).
 
-## Local development
+Recommended Ruby version: **3.1.2**
 
-Check if you have [all requirements for local environment](http://jekyllrb.com/docs/installation/).
 To install all development dependencies install [Bundler](http://bundler.io/).
 ```bash
-    gem install bundler
+gem install bundler
 ```
-and run next command from root folder:
 
+and run next command from root folder:
 ```bash
-  bundle install
+bundle install
 ```  
 
 To start Jekyll run:
 ```bash
-    jekyll serve -w
+jekyll serve -w --baseurl /test
 ```
-Site will be available at http://127.0.0.1:4000/zeppelin/ or http://localhost:4000/zeppelin/ (on Windows)
+
+Site will be available at http://127.0.0.1:4000/test/ or http://localhost:4000/test/ (on Windows)
 
 **NOTE:** in this mode all changes to html and data files will be automatically regenerated, but after changing ```_config.yml``` you have to restart server.
 
-### Sass(Compass) support
+
+### Sass (Compass) support ###
 **Note:** You need to install [Node.js](http://nodejs.org/download/)
 
 To watch changes of `.sass` files and compile it to the `.css` on a fly change property `safe: true` to `safe: false` in `_config.yml`.
 **Note: It works only on local machine, because GitHub runs Jekyll in `--save` [mode](https://help.github.com/articles/using-jekyll-with-pages/#configuration-overrides)**
 
-Learn more about Sass development from [documentation](https://github.com/gdg-x/zeppelin/wiki/Sass-development).
+Learn more about Sass development from this [documentation](https://github.com/gdg-x/zeppelin/wiki/Sass-development) by the GDG team.
 
 
-### Resource optimizations (optional)
+### Resource optimizations (optional) ###
```

Comment:
Did you mean to add the ### at the end? Just checking

Reply:
Yes, just a personal preference.